src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,143 @@ JSON example:
+|  BackgroundSuppressionNumberPulses    | OPTIONAL, RECOMMENDED if `BackgroundSuppression` is `true`  | [number][]  | The number of background suppression pulses used. Note that this excludes any effect of background suppression pulses applied before the labeling. |
+|  BackgroundSuppressionPulseTime     | OPTIONAL, RECOMMENDED if `BackgroundSuppression` is `true`  | [array][] of [numbers][]   | Array of numbers containing timing, in seconds, of the background suppression pulses before the start of the readout. In case of multi-PLD with different background suppression pulse times, only the pulse time of the first PLD should be defined. |
+|  VascularCrushingVenc   | OPTIONAL, RECOMMENDED if `VascularCrushing` is `true`  | [number][] or [array][] of [numbers][] | The crusher gradient strength, in centimeters per second. Specify either one number for the total time-series, or provide an array of numbers, for example when using QUASAR, using the value zero to identify volumes for which `VascularCrushing` was turned off. Corresponds to DICOM Tag 0018,925A `ASL Crusher Flow Limit`. |
+|  LabelingOrientation          | RECOMMENDED  | [array][] of [numbers][]    | Orientation of the labeling plane (`(P)CASL`) or slab (`PASL`). The direction cosines of a normal vector perpendicular to the ASL labeling slab or plane with respect to the patient. Corresponds to DICOM Tag 0018,9255 `ASL Slab Orientation`.|
+|  LabelingDistance    | RECOMMENDED  | [number][]  | Distance from the center of the imaging slab to the center of the labeling plane (`(P)CASL`) or the leading edge of the labeling slab (`PASL`), in millimeters. If the labeling is performed inferior to the isocenter, this number should be negative. Based on DICOM macro C.8.13.5.14. |
+|  LabelingLocationDescription     | RECOMMENDED   | [string][]    | Description of the location of the labeling plane (`(P)CASL`) or the labeling slab (`PASL`) that cannot be captured by fields ‘LabelingOrientation’ or ‘LabelingDistance’. May include a link to an anonymized screenshot of the planning of the labeling slab/plane with respect to the imaging slab or slices `sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_labeling.jpg`. Based on DICOM macro C.8.13.5.14.  |
+| LookLocker  | OPTIONAL | [boolean][]  | Boolean indicating if a Look-Locker readout is used.| 
+|  LabelingEfficiency     | OPTIONAL   | [number][]  | Labeling efficiency, specified as a number between zero and one. |
+
+### (P)CASL-specific metadata fields
+These fields can only be used when `LabelingType` is `CASL` or `PCASL`.
+| **Key name**                  | **Requirement level** | **Data type** | **Description**                                                   |
+|-------------------------------|-----------------------|---------------|------------------------------------------------------------------------------------------|
```

Comment:
This table appears malformed. It doesn't render in the RTD CI.

Reply:
fyi @patsycle @HenkMutsaerts "RTD CI" means "Readthedocs continuous integration service" and Taylor refers to the following GitHub check listed at the end of this PR (see screenshot):

![image](https://user-images.githubusercontent.com/9084751/97070524-484ca080-15d9-11eb-918f-b8567919805f.png)

by clicking on it, you can see the draft rendering of the PR, like [here](https://bids-specification--652.org.readthedocs.build/en/652/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#pcasl-specific-metadata-fields). --> There you can see that some table formatting seems to have gone awry.

Note that after each commit, you have to click again on the check at the end of the PR to get the up to date link

Reply:
Thanks, yes, that shouldn't be a problem to adjust.

Reply:
Table is fixed, but I still get some errors on a link. However, I cannot seem to find what the issue is. Any suggestions?

Reply:
@patsycle some of your links are not formatted in markdown style, see this example:

```markdown
[this-link]
(http://www.wont-work.net)

[this-link](http://www.wont-work
-either.net)

[but-this-link](http://www.will-work.success)
```

Reply:
That is, a link in markdown format must be on a single line